src/index.js

```diff
@@ -0,0 +1,16 @@
+import './style.css';
+import { addTodo, deleteAll, renderTodos } from './functions.js';
+
+document.querySelector('.text-input').addEventListener('keypress', (e) => {
+  if (e.key === 'Enter') {
+    addTodo(e.target.value);
+    renderTodos();
+    e.target.value = '';
+  }
+});
```

Comment:
Good job so far , Kindly refactor this function to doesn't allow an empty task entered into list.
<img width="470" alt="image" src="https://user-images.githubusercontent.com/122613936/237034576-73c34fdf-ff99-4440-b902-27d9933fbd01.png">